tests/dummy/app/templates/application.hbs

```diff
@@ -1,4 +1,7 @@
-<div local-class="application">
+<div
+  local-class="application no-focus-outline"
+  {{did-insert this.allowFocusOutlineOnTab}}
+>
```

Comment:
I'm not sure yet if I needed to use `{{will-destroy}}` modifier to remove event listeners when the user exits the app. Because the event listeners happen at the application level, I think it'd be okay to not remove them?